.github/actions/setup-python/action.yml

```diff
@@ -5,9 +5,45 @@ description: |
 
   Serves as the single place to update python versions over time across the
   project.
+inputs:
+  skip-venv-install:
```

Comment:
NIT: it might not matter based on other comments in this review, but I'd rather see this as `install-venv` rather than `skip-install`.